elide-core/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java

```diff
@@ -131,21 +135,13 @@ private Object fetchObjects(Environment context) {
 
         /* check whether current object has a parent or not */
         if (context.isRoot()) {
-            return fetchObject(context);
+            Class<?> loadClass = getRecordType(context.requestScope, context.outputType);
+            return fetchObject(context.requestScope, loadClass, context.ids, context.sort,
```

Comment:
Made this change to pass two fewer parameters to `fetchObject()`, as they were simply being propagated further down to fetch the `loadClass` from `getRecordType()`.